### PR TITLE
updated mobile onboarding

### PR DESCRIPTION
**Pull request by chrisdugne**
**Detailed purpose of the PR**

Improved onboarding view

before:
Screenshot 2022-11-28 at 12 27 25

after:
Screenshot 2022-11-25 at 11 01 52

### CODE REVIEW

**Comment by chrisdugne** [comment]
[[OR] Onboarding Review](https://trello.com/c/dRPJVxLP/2851-or-onboarding-review)

File: packages/@coorpacademy-components/src/atom/review-presentation/index.native.tsx [hunk 1]
```diff
@@ -53,12 +56,13 @@ const createStyleSheet = (theme: Theme): StyleSheetType =>
       alignItems: 'center',
       height: 54,
       borderRadius: 8,
-      backgroundColor: theme.colors.gray.extra,
+      backgroundColor: '#FAFAFA',
```
Comment (EmelineLeduc) [inline packages/@coorpacademy-components/src/atom/review-presentation/index.native.tsx:59]: `'#FAFAFA'` ===  `theme.colors.gray.extra` ?

**Review by EmelineLeduc — APPROVED** [review]

File: packages/@coorpacademy-components/src/atom/review-presentation/index.native.tsx (hunk 1 above)
Comment (EmelineLeduc) [inline packages/@coorpacademy-components/src/atom/review-presentation/index.native.tsx:59]: `'#FAFAFA'` === `theme.colors.gray.extra` ?

Comment (chrisdugne) [reply packages/@coorpacademy-components/src/atom/review-presentation/index.native.tsx:59]: tu as raison, il doit y avoir un pb avec le `theme`, je check

Comment (chrisdugne) [reply packages/@coorpacademy-components/src/atom/review-presentation/index.native.tsx]: https://github.com/CoorpAcademy/components/pull/2509/commits/a31ffe07742d7a3c52da9c09bd14a20ace63ca6d

**Pull request merged**